src/components/product-pages/common/nav/responsive.module.css

```diff
@@ -0,0 +1,21 @@
+.navDesktop {
```

Comment:
u can do this directly in chakra using their [breakpoints](https://v2.chakra-ui.com/docs/styled-system/responsive-styles) - if we need to adjust these breakpoints we can look into that, too but we should stick to them